src/mlpack/tests/bayesian_linear_regression_test.cpp

```diff
@@ -182,13 +183,11 @@ BOOST_AUTO_TEST_CASE(EqualtoRidge)
 
     // Check the predictions are close enough between ridge and our blr.
     for (size_t i = 0; i < y.size(); ++i)
-      BOOST_REQUIRE_CLOSE(blrPred[i], ridgePred[i], 1);
+      REQUIRE(blrPred[i] == Approx(ridgePred[i]).epsilon(1));
 
     // Exit once a test case has completed.
     break;
   }
 
-  BOOST_REQUIRE_LT(trial, 3);
```

Comment:
@mercierc should it be trial <=3 or trail < 3 , because it fails for 3

Reply:
You mean it fails with `train < 3`? That's strange because I can't see the test failed using the boost test framework.

Reply:
It passed @zoq , just a false positive , it worked

Reply:
@jeffin143 if you observed a false positive, let's change it to `trial <= 3` for paranoia (this will allow one more trial, so it should reduce the rate of false positive test failures).